This patch comments out tests that rely on files that are missing
from the source tarball.

--- openexr-2.2.0/IlmImfTest/testSampleImages.cpp	2015-02-25 16:19:21.565105625 +0100
+++ openexr-2.2.0/IlmImfTest/testSampleImages.cpp	2015-02-25 16:21:46.394128206 +0100
@@ -162,16 +162,6 @@ testSampleImages (const std::string&)
 	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_b44.exr",
 		       ILM_IMF_TEST_IMAGEDIR "comp_b44_piz.exr");
 
-	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwaa_v1.exr",
-		       ILM_IMF_TEST_IMAGEDIR "comp_dwaa_piz.exr");
-	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwaa_v2.exr",
-		       ILM_IMF_TEST_IMAGEDIR "comp_dwaa_piz.exr");
-
-	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwab_v1.exr",
-		       ILM_IMF_TEST_IMAGEDIR "comp_dwab_piz.exr");
-	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwab_v2.exr",
-		       ILM_IMF_TEST_IMAGEDIR "comp_dwab_piz.exr");
-
 
 	cout << "ok\n" << endl;
     }

